repository-data/webfiles/src/main/resources/site/freemarker/common/apicatalogue/api-catalogue.ftl

```diff
@@ -20,10 +20,21 @@
 <@headerBanner document />
 
 <div class="nhsd-t-grid nhsd-!t-margin-top-6 nhsd-api-catalogue">
-
     <div class="nhsd-t-row">
         <div class="nhsd-t-col-12">
             <#if document.body?has_content??>
+                <#if showAll??>
+                    <#if showAll == true>
+                        <#assign toShowAll="false">
+                    <#else>
+                        <#assign toShowAll="true">
+                    </#if>
+                </#if>
+
+                <@hst.renderURL var="showAlllink">
```

Comment:
Would be good to move the link generation closer to (right before) the place where it's being used.

Reply:
The `showAlllink` wants to be capitalised as `showAllLink` I think.